tests/dummy/app/components/contrib-manager/template.hbs

```diff
@@ -0,0 +1,54 @@
+<h2> Add contributor </h2>
+<div class="form">
+    <div class="form-group">
+        <label> User ID </label> {{input value=newContributorId}}
+    </div>
+    <div class="form-group">
+        <label> Permissions: </label>
+        <div class="input-group">{{radio-button name="permission" value=READ checked=newContributorPermissions}} Read</div>
+        <div class="input-group">{{radio-button name="permission" value=WRITE checked=newContributorPermissions}} Read + Write</div>
+        <div class="input-group">{{radio-button name="permission" value=ADMIN checked=newContributorPermissions}} Administrator</div>
+    </div>
+    <div class="form-group">
+        <label> Bibliographic: </label>
+        <div class="input-group">{{radio-button name="bibliographic" value=true checked=newContributorIsBibliographic}} Bibliographic </div>
+        <div class="input-group">{{radio-button name="bibliographic" value=false checked=newContributorIsBibliographic}} Non-Bibliographic </div>
+    </div>
+    <div class="form-group">
+        <button {{action 'addContributor' newContributorId newContributorPermissions newContributorIsBibliographic}} class="btn btn-primary">Add</button>
+    </div>
+</div>
+
+<table class="table">
+    <tr>
+        <th>ID</th>
+        <th> Name </th>
+        <th> Permissions </th>
+        <th> Bibliographic </th>
+        <th> </th>
+    </tr>
+    {{#each contributors as |contrib|}}
+        <tr>
+            <td>
+                {{contrib.id}}
+            </td>
+            <td>
+                {{if contrib.unregisteredContributor contrib.unregisteredContributor contrib.users.fullName}}
+            </td>
+            <td>
+                <select onchange={{action (action 'permissionChange' contrib) value='target.value'}} >
```

Comment:
Right. As far as I understand, the `value` keyword always re-writes the first argument. At the same time, having a non-keyword argument (`contrib`) after a keyword argument (`value=`) is invalid syntax. So the only way I can tell to get a reference to the contributor is to use the approach.This could be translated to:

```javascript
var innerAction = permissionChange.bind(null, contrib);
select.onchange = function(target) {
    innerAction(target.value)   
}
```